core/sql/exp/exp_clause_derived.h

```diff
@@ -1542,7 +1542,9 @@ enum conv_case_index {
   CONV_UTF8_F_UCS2_V                   =248,
 
   CONV_BLOB_BLOB                       =249,
-  CONV_BLOB_ASCII_F                    =250
+  CONV_BLOB_ASCII_F                    =250,
+
+  CONV_GBK_F_UTF8_V                    =251
```

Comment:
That may come when we support the CHAR datatype in Hive? Since right now we only support STRING in Hive, maybe we never have any CHAR columns in a Hive table?